mousestyles/dynamics/__init__.py

```diff
@@ -337,3 +337,92 @@ def mcmc_simulation(mat_list, n_per_int):
             simu_result[index] = int(state_j)
     simu_result = np.array(simu_result, dtype=int)
     return simu_result
+
+
+def get_score(true_day, simulated_day, weight=[1, 10, 50, 1]):
+    r"""
+    Returns the evaluation score for the simulted day
+    that will be later used to choose the best time
+    interval for different strains. The input data
+    should be two numpy arrays and one list, the two arrays
+    are possibly with different lengths, one being the
+    activities for one particular day of one particular
+    mouse, the other array is our simulated day for this
+    mouse from the mcmc_simulation function. And the list
+    is the weight for different status. We should give different
+    rewards for making correct simulations on various status.
+    The output will be one number between 0 and max(weight),
```

Comment:
Yeah, sure. It's because we assign weights to different status. For example, if you get a string of 5 drinks, and your simulation gives the result of 5 drinks, too, then the score would be calculated as 50*5 for the nominator, 1*5 for the denominator, which gives 50. But usually this case doesn't happen, the string we get usually contains a lot more IS and other status, normally the score will still be around 0-1.

I will indicate the datatype in the doc string